src/docker/base/Dockerfile

```diff
@@ -72,6 +72,12 @@ RUN ln -rsv "/home/lind/lind_project" "./"
 WORKDIR /root/build/
 RUN lesskey
 
+USER lind
+WORKDIR /home/lind/build/downgrade/
+RUN makepkg --noconfirm -sci
+USER root
```

Comment:
It might be fine to make these all root, just to simplify and have less user switching, but that's not a big deal

Reply:
Cool.